compiler/packages/babel-plugin-react-compiler/src/__tests__/fixtures/compiler/inline-single-return-jsx/inline-single-return-jsx.js

```diff
@@ -0,0 +1,9 @@
+// @enableInlineSingleReturnJSX
+function Component({a, b}) {
+  const c = [a, b];
+  return (
+    <Child value={c}>
```

Comment:
Btw a deopt case to think about is if `Child` is passed a reactive key, since this should remount the component when it changes right?

Reply:
good call, yeah similar to the reactive tag check.